CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# OCA Guidelines
+
+Please follow the official guide from the
+[OCA Guidelines page](https://odoo-community.org/page/contributing).
+
+## Project Specific Guidelines
+
+<!-- /!\ do not modify above this line -->
+
+This project does not have specific coding guidelines.
```

Comment:
Since this is now an org-level file, this section about project-specific guidelines does not make sense anymore.